odoo_module_migrate/migration_scripts/migrate_100_110.py

```diff
@@ -46,8 +46,12 @@
             "report.external_layout_footer is obsolete.",
     },
     ".xml": {
-        "<tree(\n|.|\t)*color=\"[^>]*":
-        "color attribute is deprecated in tree view. Use decoration- instead.",
+        "<tree[ \n\t][^>]*colors=":
+        "colors attribute is deprecated in tree view."
+        " Use decoration- instead.",
+        "<tree[ \n\t][^>]*fonts=":
```

Comment:
```suggestion
        r"<tree[\s][^>]*colors=":
        "colors attribute is deprecated in tree view."
        " Use decoration- instead.",
        r"<tree[\s][^>]*fonts=":
```

`\s` matches any whitespace:

```py
>>> re.findall(r"\s", """
...     ")
... """)
['\n', ' ', ' ', '\t', '\n']
```